src/client/features/view/components/tooltips/generateContent.js

```diff
@@ -0,0 +1,238 @@
+    h('div.field-name', key + ': '),
+    makeList(pair[1])
+  ]);
+};
+
+const metaDataKeyMap = new Map()
+  .set('Standard Name', standardNameHandler)
+  .set('Display Name', displayNameHandler)
+  .set('Data Source', dataSourceHandler)
+  .set('Type', typeHandler)
+  .set('Names', nameHandler)
+  .set('Database IDs', databaseHandler);
+
+
+//Generate HTML elements for a Parsed Metadata Field
+//Optional trim parameter indicates if the data presented should be trimmed to a reasonable length
+//Data Pair -> HTML
+function parseMetadata(pair, trim = true) {
+  let key = pair[0];
+  let handler = metaDataKeyMap.get(key);
+  if (handler) {
+    return handler(pair, trim, key);
+  }
+  else {
+    return defaultHandler(pair, trim, key);
+  }
+}
+
+//Convert a generic array or string to an html list
+//Array -> HTML 
+function makeList(items, ulClass = 'ul.value-list', liClass = 'li.value-list-item') {
+  //Delete duplicates
+  items = deleteDuplicatesWithoutCase(items);
+
+  //Resolve possible errors
+  if (typeof items === 'string') {
+    return h('div.tooltip-value', items);
+  }
+  else if (items.length === 1) {
+    return h('div.tooltip-value', items[0]);
+  }
+  else if (!(items)) {
+    return '-';
+  }
+
+  //Render List
+  return h(ulClass, items.map(item => h(liClass, item)));
+}
+
+//Render a standard tooltip item
+//Strings -> HTML
+function makeTooltipItem(value, field) {
+  return h('div.fake-paragraph', [h('div.field-name', field), h('div.tooltip-value', value.toString())]);
+}
+
+//Delete duplicates and ignore case
+//Requires a valid array
+//Array -> Array 
+function deleteDuplicatesWithoutCase(list) {
+  if (!(list instanceof Array)) {
+    return list;
+  }
+  return list.reduce((result, element) => {
+    var normalize = function (x) { return typeof x === 'string' ? x.toLowerCase() : x; };
+    var normalizedElement = normalize(element);
+    if (result.every(otherElement => normalize(otherElement) !== normalizedElement)) {
+      result.push(element);
+    }
+    return result;
+  }, []);
+}
+
+
+//Generate a data source link based on database name
+//Requires a valid database name
+//Note : prefix is optional
+//String -> HTML 
+function generateDataSourceLink(name, prefix = '') {
+  let db = config.databases;
+  let link = db.filter(value => name.toUpperCase().indexOf(value[0].toUpperCase()) !== -1);
+  if (link.length === 1 && link[0][1]) {
+    return h('div', h('div.field-name', prefix), h('a.plain-link', { href: link[0][1], target: '_blank' }, link[0][0]));
+  }
+  else if (link.length === 1) {
+    return h('div', h('div.field-name', prefix), h('a.plain-link', link[0][1]));
+  }
+  else {
+    return h('div', h('div.field-name', prefix), h('a.plain-link', name));
+  }
+}
+
+//Sort Database ID's by database name
+//Requires a valid database ID array
+//Array -> Array 
+function sortByDatabaseId(dbArray) {
+  let databases = {};
+
+  //Group Ids by database
+  for (let i = 0; i < dbArray.length; i++) {
```

Comment:
https://lodash.com/docs/4.17.4#groupBy

Reply:
Used _.GroupBy to sort the  values and then _.forEach to format the output